pandas/core/base.py

```diff
@@ -875,7 +875,7 @@ def _map_values(self, mapper, na_action=None):
                 # we specify the keys here to handle the
                 # possibility that they are tuples
                 from pandas import Series, Index
-                index = Index(mapper, tupleize_cols=False)
```

Comment:
can you add a test that hits this (IOW that makes you change this).

Reply:
[Already there](https://github.com/pandas-dev/pandas/blob/master/pandas/tests/series/test_apply.py#L418) (notice here I'm just reverting [my previous PR](https://github.com/pandas-dev/pandas/pull/18514/files#diff-9c718a39eb63c1bfac4fbeacf2906ffdR878) for consistency with the new default behaviour)... shall I mention this issue in that test?

Reply:
oh, then add this issue number there as well